### DISCUSSION TITLE
2023-09-13 Weekly Meeting 124 Notes/Agenda

### DISCUSSION THREAD

**Discussion by aaronfranke** [omigroup/omigroup#417] [Weekly Meetings]
Weekly Meeting 124 is happening on Wednesday, September 13, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in the :speaker: omi-weekly-meeting chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda:

* shoutout to the fresh voices

Continued:

* working group check-ins

Last week: https://github.com/omigroup/omigroup/discussions/415

**Comment by madjin** [comment]
## OMI Demo Day

Saturday 2pm EST September 30th, 2023

- Make a 5-10 presentation about what you're working on
- Will be in-world (likely hyperfy)
- Bring your own avatar (vrm) or will have some ready
- M3 will handle event logistics and AV technical help

![image](https://github.com/omigroup/omigroup/assets/32600939/8c3b9c4c-c259-4684-83e4-894dae1afc9c)
https://discord.gg/CGjRVNtD?event=1151561164245966989

## Portal Crawl

Another event to keep on radar is our next MetaTr@versal on Sept. 28th, we'll be updating this page soon with worlds info https://www.eventbrite.com/e/metatrversal-portal-crawl-iii-tickets-716246581807?aff=oddtdtcreator

**Comment by madjin** [comment]
## MSF Group
Some interesting events coming up
![Screenshot from 2023-09-13 12-16-14](https://github.com/omigroup/omigroup/assets/32600939/298d6c82-7e37-46b3-9ead-996a0021dd77)

### MSF Meeting notes

- https://hackmd.io/@xr/metaversestandards

![image](https://github.com/omigroup/omigroup/assets/32600939/25bd27fd-cdec-4c8e-a164-45375f2296ab)

![Screenshot from 2023-09-13 12-15-59](https://github.com/omigroup/omigroup/assets/32600939/0e4e5452-7567-433d-99b7-c9a10c47e710)
